File: Java/Java Fundamentals/Data types and variables/Exercise Tasks/Task2_SumDigits.java

import java.util.Scanner;
//100/100
public class Task2_SumDigits {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

String num = scanner.nextLine();
int sum = 0;
        for (int i = 0; i < num.length() ; i++) {
int currentNum = (int) num.charAt(i) - 48;

sum += currentNum;

        }

        System.out.println(sum);

    }
}

File: C#/C# Fundamentals/Data Type & Variables/Exercise Tasks/Task2_SumDigits.cs
using System;

namespace Task2_SumDigits
{
    class Task2_SumDigits
    {
        static void Main(string[] args)
        {
            String num = Console.ReadLine();
            int sum = 0;

            for (int i = 0; i < num.Length; i++)
            {
                int currentNum = (int)num[i] - 48;

                sum += currentNum;

            }

          

            Console.WriteLine(sum);
           
        }
    }
}
